ui/src/app/pages/fleet/agents/add/agent.add.component.ts

```diff
@@ -158,9 +160,15 @@ export class AgentAddComponent {
     const payload = this.wrapPayload(false);
 
     if (this.isEdit) {
-      this.agentsService.editAgent({...payload, id: this.agentID}).subscribe(resp => this.goBack());
+      this.agentsService.editAgent({...payload, id: this.agentID}).subscribe(() => {
+        this.notificationsService.success('Agent successfully updated', '');
+        this.goBack();
+      });
     } else {
-      this.agentsService.addAgent(payload).subscribe(() => this.goBack());
+      this.agentsService.addAgent(payload).subscribe(() => {
+        this.notificationsService.success('Sink successfully created', '');
```

Comment:
We need to change this message with "Agent successfully created"

Reply:
done